icu4c/source/test/intltest/dtfmttst.cpp

```diff
@@ -5231,7 +5231,7 @@ void DateFormatTest::TestDayPeriodWithLocales() {
     sdf.setTimeZone(*tz);
 
     sdf.applyPattern(UnicodeString("hh:mm:ss BBBB"));
-    assertEquals("hh:mm:ss BBBB | 01:00:00 | es_CO", UnicodeString("01:00:00 a.\\u00A0m.").unescape(),
+    assertEquals("hh:mm:ss BBBB | 01:00:00 | es_CO", UnicodeString("01:00:00 de la ma\\u00F1ana").unescape(),
```

Comment:
Optional: Use a string literal u"01:00:00 de la mañana" instead of the UnicodeString(...).unescape() thing.

Reply:
Done